frontend/src/components/ActivityReportsTable/ReportRow.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Tag, Checkbox,
+} from '@trussworks/react-uswds';
+import { Link, useHistory } from 'react-router-dom';
+
+import ContextMenu from '../ContextMenu';
+import { getReportsDownloadURL } from '../../fetchers/helpers';
+import TooltipWithCollection from '../TooltipWithCollection';
+
+function ReportRow({
+  report, openMenuUp, handleReportSelect, isChecked,
+}) {
+  const {
+    id,
+    displayId,
+    activityRecipients,
+    startDate,
+    author,
+    topics,
+    collaborators,
+    lastSaved,
+    calculatedStatus,
+    legacyId,
```

Comment:
Would it be better to make the API for generating the report more generic, and an instance of the API have the required specifics?

Reply:
Yeah it would be nice to have some way of defining which fields are required for the table be passed in as props. We will need _something_ like that when we update the "My Alerts" table to use this component. However in this PR I'm trying to keep changes to a minimum, only want to move the table to a new component, not make any substantial changes.